### PR TITLE
Scrutinizer-CI is not used

### PR DESCRIPTION
**Pull request by Maks3w**
Initial experiment is 2 years old

https://scrutinizer-ci.com/g/justinrainbow/json-schema/inspections

### CODE REVIEW

File: .scrutinizer.yml [hunk 1]
```diff
@@ -1,5 +0,0 @@
-before_commands:
```
Comment (GrahamCampbell) [inline .scrutinizer.yml]: Scrutiniser doesn't allow you to run this command anyway anymore.

Comment (bighappyface) [inline .scrutinizer.yml]: @GrahamCampbell this PR is to remove the Scrutinizer file from the repo.

**Comment by bighappyface** [comment]
+1

File: .scrutinizer.yml (hunk 1 above)
Comment (GrahamCampbell) [inline .scrutinizer.yml]: I know. That's why I said "anyway anymore.".

Comment (GrahamCampbell) [inline .scrutinizer.yml]: I was pointing out that this config is wrong anyway.

**Pull request merged**